provider/fs/benches/provider_fs.rs

```diff
@@ -68,7 +66,6 @@ fn json_bench(c: &mut Criterion) {
     });
 }
 
-#[cfg(all(feature = "bench", feature = "deserialize_bincode_1"))]
```

Comment:
This should still be gated on `bench`, also below.

Reply:
Thanks for catching this, fixed 👍